relays/ethereum/src/ethereum_sync_loop.rs

```diff
@@ -177,16 +172,16 @@ impl TargetClient<EthereumHeadersSyncPipeline> for SubstrateHeadersTarget {
 	}
 
 	fn submit_headers(self, headers: Vec<QueuedEthereumHeader>) -> Self::SubmitHeadersFuture {
-		let (signer, sign_transactions) = (self.signer, self.sign_transactions);
-		substrate_client::submit_ethereum_headers(self.client, signer.clone(), headers, sign_transactions)
+		let (sign_transactions, sign_params) = (self.sign_transactions, self.sign_params);
```

Comment:
```suggestion
		let { sign_transactions, sign_params, client, ... } = self;
```
This should work too.